### PR TITLE
[BugFix] Fix collectAllDatabase miss lock query statement db in insert statement

### PR DESCRIPTION
**Pull request by HangyuanLiu**
## What type of PR is this：
- [x] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] I have added user document for my new feature or new function

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
starrocks_be_unittest succeeded.

**Review by kangkaisen — DISMISSED** [review]

**Review by Seaven — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by wanpengfei-git** [comment]
starrocks_be_unittest succeeded.

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 3 / 3 (100.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/analyzer/AnalyzerUtils.java|3|3|100.00%|[]|

File: fe/fe-core/src/main/java/com/starrocks/sql/analyzer/AnalyzerUtils.java [hunk 1]
```diff
@@ -150,6 +151,12 @@ public Void visitUpdateStatement(UpdateStmt node, Void context) {
             return null;
         }
 
+        @Override
+        public Void visitDeleteStatement(DeleteStmt node, Void context) {
+            getDB(node.getTableName());
```
Comment (packy92) [inline fe/fe-core/src/main/java/com/starrocks/sql/analyzer/AnalyzerUtils.java:156]: If we support delete/update statements references multiple tables, there shoud be also changed. Can you add a TODO or comment on the UpdateStmt/DeleteStmt.class to hlep people remember change this class at the same time.

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 3 / 3 (100.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/sql/analyzer/AnalyzerUtils.java|3|3|100.00%|[]|

**Review by kangkaisen — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
starrocks_be_unittest succeeded.

**Comment by HangyuanLiu** [comment]
run starrocks_admit_test

File: fe/fe-core/src/main/java/com/starrocks/sql/analyzer/AnalyzerUtils.java (hunk 1 above)
Comment (HangyuanLiu) [reply fe/fe-core/src/main/java/com/starrocks/sql/analyzer/AnalyzerUtils.java:158]: done

**Review by kangkaisen — APPROVED** [review]

**Review by packy92 — APPROVED** [review]

**Review by Seaven — APPROVED** [review]

**Pull request merged**